### PR TITLE
Preparing release 2.0.0

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,27 +3,35 @@ Changelog
...
 This document describes changes between each past release.
 
-1.6.0 (unreleased)
+1.6.0 (2017-07-05)
 ------------------
 
-- Nothing changed yet.
+**Internal changes**
+
+- Upgrade to autograph 2.0
```
Comment (leplatrem) [inline CHANGELOG.rst]: I'm sorry because I know you told me already: but is autograph v2 backward compatible with v1?

Comment (Natim) [inline CHANGELOG.rst]: If by backward compatible you mean it doesn't break clients then yes.

Comment (leplatrem) [inline CHANGELOG.rst]: Does kinto-signer will continue to work with autograph v1 servers?

Comment (Natim) [inline CHANGELOG.rst]: I don't think so.

Comment (Natim) [inline CHANGELOG.rst]: Let's make it a breaking change then.

**Comment by leplatrem** [comment]
Can you mention that autograph v2 is required in the README?

**Comment by Natim** [comment]
Yes sure.

**Pull request merged**